scripts/install-worker.sh

```diff
@@ -119,50 +123,39 @@ sudo mkdir -p /etc/eks
 sudo mv $TEMPLATE_DIR/iptables-restore.service /etc/eks/iptables-restore.service
 
 ################################################################################
-### Docker #####################################################################
+### installing updated cli #####################################################
+### https://docs.aws.amazon.com/cli/v1/userguide/install-linux.html#install-linux-bundled
 ################################################################################
 
-sudo yum install -y device-mapper-persistent-data lvm2
-
-INSTALL_DOCKER="${INSTALL_DOCKER:-true}"
-if [[ "$INSTALL_DOCKER" == "true" ]]; then
-    sudo amazon-linux-extras enable docker
-    sudo groupadd -og 1950 docker
-    sudo useradd --gid $(getent group docker | cut -d: -f3) docker
-
-    # install runc and lock version
-    sudo yum install -y runc-${RUNC_VERSION}
-    sudo yum versionlock runc-*
-
-    # install containerd and lock version
-    sudo yum install -y containerd-${CONTAINERD_VERSION}
-    sudo yum versionlock containerd-*
-
-    # install docker and lock version
-    sudo yum install -y docker-${DOCKER_VERSION}*
-    sudo yum versionlock docker-*
-    sudo usermod -aG docker $USER
-
-    # Remove all options from sysconfig docker.
-    sudo sed -i '/OPTIONS/d' /etc/sysconfig/docker
-
-    sudo mkdir -p /etc/docker
-    sudo mv $TEMPLATE_DIR/docker-daemon.json /etc/docker/daemon.json
-    sudo chown root:root /etc/docker/daemon.json
-
-    # Enable docker daemon to start on boot.
-    sudo systemctl daemon-reload
+if [[ "$BINARY_BUCKET_REGION" != "us-iso-east-1" && "$BINARY_BUCKET_REGION" != "us-isob-east-1" ]]; then
```

Comment:
not sure if there will have any exception, how about...

```
if echo ${BINARY_BUCKET_REGION} | grep -q -E 'us-iso[b]?-east-1'; then
  ...
fi
```

Reply:
I kind of like spelling these out here. It's a little clearer to me since it's just two.